app/controllers/proposed_organisations_controller.rb

```diff
@@ -8,10 +8,9 @@ def index
   end
 
   def update
-    proposed_org = ProposedOrganisation.friendly.find params[:id]
-    result = AcceptProposedOrganisation.new(proposed_org).run
-    set_flash_for_accepting_proposed_org result
-    redirect_to organisation_path(result.accepted_organisation) and return false
+    rslt = prepare_data_for_update
+    redirect_to organisation_path(rslt.accepted_org) and return false if rslt.accepted_org
+    redirect_to proposed_organisations_path
```

Comment:
Yes, of course, I understand. However, it will add two additional CC issues (6 instead of 5 allowed lines of code in one method & ABC complexity). It is not a problem to change these lines, but one of first comments on my PR was about necessity to fix CodeClimate issues.